packages/e2e-test-utils-playwright/src/test.ts

```diff
@@ -93,14 +98,25 @@ function observeConsoleLogging( message: ConsoleMessage ) {
 		return;
 	}
 
+	// https://bugzilla.mozilla.org/show_bug.cgi?id=1404468
+	if (
+		text.includes( 'Layout was forced before the page was fully loaded' )
+	) {
+		return;
+	}
+
+	// Deprecated warnings coming from the third-party libraries.
```

Comment:
Can we mention specifically which ones?

Reply:
Unfortunately, I wasn't able to track it down in the library. Event full search in the directory returns nothing.